src/main/java/com/android/volley/Request.java

```diff
@@ -56,6 +56,12 @@
         int PATCH = 7;
     }
 
+    /** Callback interface to notify listeners when a request has been canceled. */
+    public interface CancelListener {
```

Comment:
The names seems odd. Ideas for a better name:

* OnRequestCanceledListener
* CancellationListener
* OnCancelListener

Reply:
CancellationListener makes sense - the intent is to be consistent with "ErrorListener", so the prefixes should both be nouns describing what happened.